pynecone/components/component.py

```diff
@@ -370,7 +378,11 @@ def render(self) -> str:
         tag = self._render()
         return str(
             tag.add_props(
-                **self.event_triggers, key=self.key, sx=self.style, id=self.id
+                **self.event_triggers,
+                key=self.key,
+                sx=self.style,
+                id=self.id,
+                className=self.class_name,
```

Comment:
I think you can have the key here as `class_name` - it should get converted to `className` during compilation

Reply:
Right! I changed this and worked.